src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/ProvidePublicParameterlessSafeHandleConstructor.cs

```diff
@@ -71,11 +75,22 @@ private static void AnalyzeSymbol(SymbolAnalysisContext context, INamedTypeSymbo
 
                     // If a parameterless constructor has been defined, emit the diagnostic on the constructor instead of on the type.
                     targetWarningSymbol = constructor;
+                    diagnosticPropertyBag = diagnosticPropertyBag.Add(DiagnosticPropertyConstructorExists, null);
```

Comment:
💡 It may be good to set the values here to `""` instead of `null`. I'm not 100% sure how `null` gets propagated through the whole system, but I'm sure about `""` so it leads to fewer questions.